calico-enterprise/operations/ebpf/enabling-ebpf.mdx

```diff
@@ -160,6 +163,13 @@ To re-enable it:
 kubectl patch networks.operator.openshift.io cluster --type merge -p '{"spec":{"deployKubeProxy": true}}'
 ```
 
+#### MKE
+
+If you are running MKE, you can disable `kube-proxy` as follows:
+
+Download the cluster configuration, set `kube_proxy_mode` to `disabled` and `kube_default_drop_masq_bits` to `true` and upload the cluster configuration.
```

Comment:
"Download the cluster configuration" is a bit vague, can you either make this a link to the MKE docs or give the command to do it?